app/docs/enterprise/0.31-x/kong-implementation-checklist.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Kong Implementation Checklist
+---
+# Kong Implementation Checklist
+
+- Kong can run on instances of any size with any resources. While the system requirements vary significantly depending on the use-case, generally speaking, we recommend to start big and then gradually reduce the instances to the appropriate number and size.
+- When allocating resources to a Kong cluster, slightly over-provision the cluster to make room for request spikes. Kong requires a database to run and you can choose either Cassandra or Postgres.
+
+### Postgres
+
+- Generally speaking, Postgres works well for most of the use-cases, and it’s usually easier to setup. We recommend setting up a master-slave replication with servers located in different racks, data centers or availability zones to account for infrastructure failures.
+- In multi-DC environments, Postgres may not be the right fit, since a Kong node located in a different datacenter will have to send write requests all the way to the Postgres data-center, adding increased latency to the system.
+
+### Cassandra
+
+- We recommend using Cassandra in multi-DC environments because it supports native replication and availability capabilities of the system. When starting a Cassandra cluster for Kong, we recommend starting at least 3 nodes in every datacenter with a replication setting of 2.
+- Cassandra is an eventually consistent datastore, which means that over time the data will be the same across the cluster, so please account for inconsistencies in the system.
+
+### Kong Cluster
+
+- Regardless of the datastore being adopted with Kong, the Kong nodes themselves need to join in a cluster. The Kong nodes will talk to each other via their connections to the database (Postgres or Cassandra). Please refer to the [clustering reference](/docs/latest/clustering/) for more details.
+
+## Production System Requirements
+
+- For Kong prefer CPU optimized instances. We recommend at least 4 CPU cores, 16GB of RAM and 24GB of disk space. Kong performs better on a system with multiple cores. In an AWS environment, we recommend running c 3.2xlarge instances in production that guarantee enough cores (8) and memory (16GB), with a local storage instead of EBS to improve the performance.
+- For Cassandra and Postgres prefer memory optimized instances. Assume more intensive workload on the datastore if you are using the rate-limiting or response rate-limiting plugins with a “cluster” policy.
+High ulimit systemvalue, possibly >=65535 . A higher limit value will allow Kong and the datastores to process more incoming requests.
+
+## Kong High Availability
```

Comment:
maybe just "High Availability" - I wonder what else we could be talking about here :thinking: